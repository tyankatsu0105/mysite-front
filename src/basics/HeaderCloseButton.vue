<template>
	<button
		class="HeaderCloseButton"
		@click="closeHeader"
	>
		<font-awesome-icon icon="times" />
	</button>
</template>

<script>
export default {
  methods: {
    closeHeader() {
      this.$emit("close-header", false);
    }
  }
};
</script>

<style scoped lang="scss">
.HeaderCloseButton {
  position: absolute;
  top: 5px;
  right: 5px;
  font-size: 1.5rem;
  color: inherit;
}
</style>
