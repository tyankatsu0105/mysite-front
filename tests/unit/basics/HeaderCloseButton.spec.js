import { shallowMount } from "@vue/test-utils";
import HeaderCloseButton from "@/basics/HeaderCloseButton.vue";

describe("HeaderCloseButton", () => {
  it("render", () => {
    const wrapper = shallowMount(HeaderCloseButton);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
