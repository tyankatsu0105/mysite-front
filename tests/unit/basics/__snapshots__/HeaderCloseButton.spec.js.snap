// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderCloseButton render 1`] = `
<button class="HeaderCloseButton">
  <fontawesomeicon-stub icon="times"></fontawesomeicon-stub>
</button>
`;
